paasta_tools/kubernetes_tools.py

```diff
@@ -1278,19 +1279,26 @@ def format_kubernetes_app(self) -> Union[V1Deployment, V1StatefulSet]:
                     ),
                 )
 
+            prometheus_shard = self.get_prometheus_shard()
+            if prometheus_shard:
+                complete_config.metadata.labels[
+                    "paasta.yelp.com/prometheus_shard"
+                ] = prometheus_shard
+
+            prometheus_path = self.get_prometheus_path()
```

Comment:
this is here twice?

Reply:
I don't understand. One is the original `prometheus_shard` variable and the second is a new variable/label called `prometheus_path` (see .rst file change for description.